mitosheet/mitosheet/code_chunks/postprocessing/set_dataframe_format_code_chunk.py

```diff
@@ -290,9 +290,16 @@ def get_dataframe_format_code(state: State, sheet_index: int) -> Tuple[Optional[
 class SetDataframeFormatCodeChunk(CodeChunk):
     """
     This is the first postprocessing code chunk. Postprocessing code chunks are executed after all other code chunks,
-    and are given access to the final state. They have no parameters otherwise.
+    and are given access to the final state. This means, unlike other code chunks, it's fine for post-processing code 
+    chunks to get access to the final post_state. 
+    
+    TODO: Maybe they should only get the post state?
```

Comment:
There could be a parent class `CodeChunk` and children classes like `PostProcessingCodeChunk` and `PreProcessingCodeChunk` that would guide people to do the right thing here